components/web/goaccess/Makefile

```diff
@@ -11,43 +11,35 @@
 #
 # Copyright 2016 Adam Stevko
 #
+BUILD_BITS=64
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		goaccess
-COMPONENT_VERSION=	1.2
-COMPONENT_REVISION=	2	
+COMPONENT_VERSION=	1.4
 COMPONENT_SUMMARY=	GoAccess is an open source real-time web log analyzer and interactive viewer that runs in a terminal in *nix systems or through your browser.
 COMPONENT_PROJECT_URL=	https://goaccess.io/
 COMPONENT_FMRI=		web/goaccess
 COMPONENT_CLASSIFICATION=Web Services/Application and Web Servers
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_URL=	http://tar.goaccess.io/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE_HASH= sha256:6ba9f66540ea58fc2c17f175265f9ed76d74a8432eeac1182b74ebf4f2cd3414
+COMPONENT_ARCHIVE_URL=	https://tar.goaccess.io/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH= sha256:e8fbb9ff852556d72dfd9f1d0134ba075ce5b4c41289902a6f4e0d97c69377be
 COMPONENT_LICENSE=	MIT
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+include $(WS_MAKE_RULES)/common.mk
 
 COMPONENT_PRE_CONFIGURE_ACTION= \
 	$(CLONEY) $(SOURCE_DIR) $(@D)
 
 CONFIGURE_BINDIR.64= $(CONFIGURE_BINDIR.32)
 
```

Comment:
You need to remove this line. At least on my build system it is responsible for the $(MACH32) in the file action:
file path=usr/bin/$(MACH32)/goaccess